cmd/deploy/remote.go

```diff
@@ -15,111 +15,31 @@ package deploy
 
 import (
 	"context"
-	"crypto/rand"
```

Comment:
Most logic of this file was moved to `remote/remote.go` to unify logic with `deploy/remote.go`